alf/algorithms/generator.py

```diff
@@ -90,7 +180,15 @@ def __init__(self,
                  entropy_regularization=0.,
                  mi_weight=None,
                  mi_estimator_cls=MIEstimator,
-                 par_vi="gfsf",
+                 par_vi=None,
+                 critic_input_dim=None,
+                 critic_hidden_layers=(100, 100),
+                 critic_l2_weight=10.,
```

Comment:
The paper _Stochastic Stein Discrepencies_ shows that SVGD converges even in unbounded domains. Further conversation with author suggests the L2 penalty on this critic might not be needed. 
Results with generator test and BLR suggest this is true. 
I suggest leaving the l2 penalty in, to be consistent with the fisher neural sampler formulation, but possibly set the weight to 0 by default.

Reply:
Directly setting L2 penalty to be zero cannot pass current generator tests and hypernetwork tests. Maybe some further tuning is needed. We could revisit this later. I have added a comment in #751.